autopush/tests/support.py

```diff
@@ -44,5 +45,6 @@ def test_db(metrics=None):
         router_conf=DDBTableConfig(tablename='router'),
         router=Mock(spec=Router),
         message_conf=DDBTableConfig(tablename='message'),
-        metrics=SinkMetrics() if metrics is None else metrics
+        metrics=SinkMetrics() if metrics is None else metrics,
+        sessions=autopush.tests.sessions,
```

Comment:
for unknown reasons, nose doesn't import `sessions` from `autopush.tests` correctly (FWIW, pytest does). Thus the fully qualified path, which does resolve to the correct global pool.